assets/templates/dining_lists/dining_slot_info.html

```diff
@@ -5,174 +5,179 @@
+                      class="col-12 col-md-6 p-0 my-md-0 my-3">
+                    {% csrf_token %}
+                    <button type="submit" class="btn-block btn btn-outline-warning">Sign out</button>
+                </form>
+            {% elif user_can_add_self %}
+                {% url 'entry_add' day=date.day month=date.month year=date.year identifier=dining_list.association.slug as url%}
+                <form method="post"
+                      action="{{ url }}?next={{ request.path_info }}"
+                      class="col-12 col-md-6 p-0 my-md-0 my-3">
+                    {% csrf_token %}
+                    <button type="submit" class="btn-block btn btn-primary">Sign up</button>
+                </form>
+            {% else %}
+                <div class="btn-disabled btn col-12 col-md-6 ">You can not join this list</div>
+            {% endif %}
+
+            {% if user_can_add_others %}
                 {% url 'entry_add' day=date.day month=date.month year=date.year identifier=dining_list.association.slug as url %}
-                {% if user_is_on_list %}
-                    <div class="col-12 col-md-6">
-                {% else %}
-                    <div class="col-12 col-md-4">
+                <a href="{{ url }}" class="btn btn-outline-primary col-12 col-md-6 ">Add others</a>
+            {% endif %}
+        {% else %}
+            {% if user_can_add_others %}
+                {# Add other button #}
+                {% url 'entry_add' day=date.day month=date.month year=date.year identifier=dining_list.association.slug as url %}
+                <a href="{{ url }}" class="btn btn-outline-primary col-12 col-md-6 offset-md-6">Add others</a>
+            {% endif %}
+        {% endif %}
+    </div>
+
+    <hr class="col-12 my-3 p-0">
+
+    <div class="col-12 row m-0">
+        <div class="col-12 col-md-10 row px-0">
+            {% if dining_list.is_open %}
+                <div class="col-3">
+                    Open till
+                </div>
+                <div class="col-9 h5">
+                    {% if dining_list.sign_up_deadline.date == dining_list.date %}
+                        {{ dining_list.sign_up_deadline |date:"H:i" }}
+                    {% else %}
+                        {{ dining_list.sign_up_deadline |date:"l H:i" }}
+                    {% endif %}
+                </div>
+            {% else %}
+                <div class="col-9 offset-3">
+                    Dining list is closed, contact {{ dining_list.claimed_by }} if you want to join
+                </div>
+            {% endif %}
+
+            <div class="col-3 pt-3">Diners:</div>
+            <div class="col-3 pt-3">
+                <span class="h5">{{ dining_list.dining_entries.count }}</span> / {{ dining_list.max_diners }}
+            </div>
+            <div class="col-6 pt-3">
+                {% if number_of_allergies > 0 %}
+                    {{ number_of_allergies }} allerg{{ number_of_allergies|pluralize:"y,ies" }}
```

Comment:
I'd say '3 people with allergies or food wishes' instead of '3 allergies'.

Reply:
I get your point, but 'food wishes' insinuates that allergies can be used to tell preferences. This is not something you want, allergies should be really allergies otherwise the allergy list is gonna get crowded with non-allergies (I've seen it happen on the old Knights dining list). That said, there is room for a future improvements on vegas

Reply:
I've replaced it with 'has an allergy/have allergies' for now

Reply:
The more formal version is dietary wishes. Vegetarian is obviously not an allergy. That happening on the knights list was probably not due to the text, if people want to say something they'll put it down wherever they most easily can. (And we're not going to check if people actually put down allergies in the box)